### ISSUE TITLE
pct_cover automatically removes groupings that start or end with "Na"

### ISSUE THREAD

**Issue by annack84** [Landscape-Data-Commons/terradactyl#46]
The pct_cover function filters out indicators for user grouping variables that start or end with "NA". For example, I included a nativity column in my species file (filled with either "Introduced" or "Native"), and pct_cover only gave me the indicators for the "Introduced" group.

Here's the existing code (lines 168 - 172):
  summary <- summary %>% subset(!grepl(
    x = indicator,
    pattern = "^[.]|[.]$|\\.\\.|\\.NA|NA\\.|\\.NA\\."
  ))

Here's a fix that worked for me:
  summary <- summary %>% subset(!grepl(
    x = indicator,
    pattern = "^[.]|[.]$|\\.\\.|\\.NA$|^NA\\.|\\.NA\\."
  ))

**Comment by smccord** [comment]
Thanks! Can you send me a copy of your species file?

**Comment by annack84** [comment]
Sure, here it is. I discovered this issue when trying to calculate percent cover with these grouping variables: PhotosyntheticPathway, Native, Duration, GrowthHabitSub. The results I got only included the "Introduced" indicators - "Native" indicators were excluded.

[SpeciesList_SWEDD_Vernal_20210811.csv](https://github.com/Landscape-Data-Commons/terradactyl/files/7015115/SpeciesList_SWEDD_Vernal_20210811.csv)
